### ISSUE TITLE
Cannot open MLE.ipynb

### ISSUE THREAD

**Issue by frederickz** [rickecon/StructEst_W20#14]
I think there is something wrong with the latest version of `MLE.ipynb` (latest commit `0b5f18b`). 

Here is the error log by Jupyter:
```
Error loading notebook
Unreadable Notebook: /Users/yumingz/Projects/StructEst_W20/Notebooks/MLE/MLest.ipynb 
NotJSONError("Notebook does not appear to be JSON: '\\n\\n\\n\\n\\n\\n<!DOCTYPE html>\\n<html lang...")
```
Here is the error log by Visual Studio Code:
```
Unexpected token < in JSON at position 6
```

**Comment by frederickz** [comment]
Does anyone else have this problem after `fetch` and `merge` the latest commit?

**Comment by xiangyum** [comment]
Yep. Had the same issue. I could still see it off Rick's master though.

**Comment by rickecon** [comment]
@zhuyuming96 @xiangyum . I don't know what this problem is, and I can't replicate it. Maybe it is an issue with VS Code. I get the notebook to open just fine by going to my terminal and typing the following four commands:
```
git fetch upstream
git merge upstream/master
git push origin master
```
After these three commands, your remote repository will be up-to-date with the main repository. Maybe try VS Code after doing these steps. If that doesn't work, then just open the notebook from your terminal by going to the `Notebooks/MLE/` folder and typing `jupyter notebook`.
